File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1159.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
while(true){
    n = parseInt(lines.shift());
    if(n==0){
        break
    }
    if(n%2!=0){
        n+=1
    }
    soma=0
    for(i=0;i<5;i++){
        soma+=n
        n+=2
    }
    console.log(soma)
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1159.lua
while(true) do
    n = io.read("*number")
    if(n==0) then
        break
    end
    if(n%2~=0) then
        n=n+1
    end
    soma=0
    for i=1,5 do
        soma=soma+n
        n=n+2
    end
    print(soma)
end